Basic/ClientDriven/Packages/manifest.json

```diff
@@ -6,7 +6,7 @@
     "com.unity.ide.visualstudio": "2.0.16",
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.inputsystem": "1.4.3",
-    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop",
+    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#a63fde7fbdfdbd601bae9cc42d93b7e6bc9d49c8",
```

Comment:
adding a comment here to block this until it's merged boss room side.

Reply:
I'll leave your comment open until the Boss Room update has been merged.